xds/src/main/java/io/grpc/xds/sds/SecretProvider.java

```diff
@@ -23,8 +23,7 @@
  * A SecretProvider is a "container" or provider of a secret. This is used by gRPC-xds to access
  * secrets, so is not part of the public API of gRPC. This "container" may represent a stream that
  * is receiving the requested secret(s) or it could represent file-system based secret(s) that are
- * dynamic. Synchronous and Asynchronous methods to access the underlying secret are available. See
- * {@link SecretManager} for a note on lifecycle management
+ * dynamic.
```

Comment:
this is unrelated changes, probably need to be a separate PR. (or 2 commits and rebase the change)

Reply:
done